src/dags/importscripts/import_sport.py

```diff
@@ -0,0 +1,163 @@
+import csv
+from hashlib import blake2s
+import json
+import logging
+from typing import List, Tuple, TypedDict, Generator, Type
+
+
+# define logger for output to console
+logger = logging.getLogger(__name__)
+
+
+def get_dialect(file: str) -> Type:
+    """determine dialect of file
+
+    Args:
+        file: the path/to/file.ext that will be used to determine it's dialect
+
+    Returns:
+        A subclass dialect of Sniffer
+
+    """
+    with open(file) as data:
+        return csv.Sniffer().sniff(data.readline())
+
+
+def get_header(file: str) -> List:
+    """Return header row of file
+
+    Args:
+        file: the path/to/file.ext that will be used to determine it's header
+
+    Returns:
+        a list of columns in string
+
+    """
+    dialect = get_dialect(file)
+    with open(file, "r") as src:
+        data = csv.reader(src, dialect=dialect)
+        return next(data)
+
+
+def make_hash(composite_values: List, digest_size: int = None) -> int:
+    """The blake2s algorithm is used to generate a single hased value for source
+    composite values that uniquely identify a row.
+    In case the source itself doesn't provide a single solid identification as key.
+
+    Args:
+        composite_values: a concatenated string of record values based on
```

Comment:
The concatenation happens in the function body, hence the parameter is not yet concatenated.

Reply:
On second thought I'm not sure my remark is valid. After all you are concatenating things on line 160 in you call to `make_hash`

Reply:
Dit is een dubbele concatenate, heb deze over het hoofd gezien. Ik laat die in de hash functie staan.